BoxBrowseSDK/BoxBrowseSDK/BOXItemsViewController.h

```diff
@@ -58,6 +58,20 @@
 - (BOOL)itemsViewController:(BOXItemsViewController *)itemsViewController shouldEnableItem:(BOXItem *)item;
 
 /**
+ *  By default, items will be sorted for display with the following rules:
```

Comment:
Minor description changes:
"Override this method to enable item sorting. This method sorts BOX folder item results for display.

@param itemsViewController The instance of BOXItemsViewController calling this method

@return YES to enable default sorting:
1) Folders listed before files and weblinks
2) By modified date descending
3) Alpha-numeric ascending.
@return YES and implement 
   itemsViewController:compareForSortingItem:toItem:
   for custom sorting.
@return NO to disable sorting.